TS2307: Cannot find module 'puppeteer' or its corresponding type declarations

**Environment (please complete the following information):**
 - **WebdriverIO version:** 6.11.0

**Describe the bug**
Error TS2307: Cannot find module 'puppeteer' or its corresponding type declarations.

**To Reproduce**
Steps to reproduce the behavior:

1.Use TypeScript;
2.Execute tsc;

**Actual Result**
node_modules/webdriverio/webdriverio-core.d.ts:596:44 - error TS2307: Cannot find module 'puppeteer' or its corresponding type declarations.
596     type PuppeteerBrowser = Partial<import('puppeteer').Browser>;
                                              
node_modules/webdriverio/webdriverio-core.d.ts:597:38 - error TS2307: Cannot find module 'puppeteer' or its corresponding type declarations.
597     type CDPSession = Partial<import('puppeteer').CDPSession>;

--- a/packages/webdriverio/package.json
+++ b/packages/webdriverio/package.json
@@ -74,7 +74,7 @@
     "lodash.zip": "^4.2.0",
     "minimatch": "^3.0.4",
     "puppeteer-core": "^5.1.0",
-    "@types/puppeteer-core": "^2.0.0"
+    "@types/puppeteer-core": "^5.4.0",
     "resq": "^1.9.1",
     "rgb2hex": "0.2.3",
     "serialize-error": "^7.0.0",

--- a/packages/webdriverio/package.json
+++ b/packages/webdriverio/package.json
@@ -53,8 +53,7 @@
     "@types/lodash.clonedeep": "^4.5.6",
     "@types/lodash.isobject": "^3.0.6",
     "@types/lodash.isplainobject": "^4.0.6",
-    "@types/lodash.zip": "^4.2.6",
-    "@types/puppeteer-core": "^2.0.0"
+    "@types/lodash.zip": "^4.2.6"
   },
   "dependencies": {
     "@wdio/config": "6.11.0",
@@ -75,6 +74,7 @@
     "lodash.zip": "^4.2.0",
     "minimatch": "^3.0.4",
     "puppeteer-core": "^5.1.0",
+    "@types/puppeteer-core": "^2.0.0"
     "resq": "^1.9.1",
     "rgb2hex": "0.2.3",
     "serialize-error": "^7.0.0",

